server/src/main/java/org/elasticsearch/search/aggregations/support/ValuesSourceConfig.java

```diff
@@ -195,7 +195,7 @@ public ValuesSourceConfig script(AggregationScript.LeafFactory script) {
         return this;
     }
 
-    public ValuesSourceConfig scriptValueType(ValueType scriptValueType) {
+    private ValuesSourceConfig scriptValueType(ValueType scriptValueType) {
```

Comment:
This is unrelated, but I noticed `scriptVauleType` was only ever used internaly while doing the rest of this change,   and the scope restriction is small.